packages/account/src/account.ts

```diff
@@ -82,11 +82,11 @@ class Chain0Reader implements commons.reader.Reader {
     return undefined
   }
 
-  async nonce(_wallet: string, _space: ethers.BigNumberish): Promise<ethers.BigNumberish> {
-    return ethers.constants.Zero
+  async nonce(_wallet: string, _space: BigIntish): Promise<bigint> {
```

Comment:
ethers still has the `ethers.BigNumberish` type, but now it uses native bigInt or strings ..

Our response type is now `Promise<bigint>` ... what is `bigint` ..? Im just wondering what is most sensible return type..